src/gui/org/deidentifier/arx/gui/view/impl/define/ViewAttributeList.java

```diff
@@ -225,16 +238,20 @@ private void create(final Composite parent) {
         column0.setWidth("5%", "25px"); //$NON-NLS-1$ //$NON-NLS-2$
         DynamicTableColumn column1 = new DynamicTableColumn(table, SWT.NONE);
         column1.setText(Resources.getMessage("ViewAttributeList.0")); //$NON-NLS-1$
-        column1.setWidth("45%", "30px"); //$NON-NLS-1$ //$NON-NLS-2$
+        column1.setWidth("20%", "30px"); //$NON-NLS-1$ //$NON-NLS-2$
         DynamicTableColumn column2 = new DynamicTableColumn(table, SWT.NONE);
         column2.setText(Resources.getMessage("ViewAttributeList.1")); //$NON-NLS-1$
-        column2.setWidth("25%", "30px"); //$NON-NLS-1$ //$NON-NLS-2$
+        column2.setWidth("20%", "30px"); //$NON-NLS-1$ //$NON-NLS-2$
         DynamicTableColumn column3 = new DynamicTableColumn(table, SWT.NONE);
         column3.setText(Resources.getMessage("ViewAttributeList.2")); //$NON-NLS-1$
-        column3.setWidth("25%", "30px"); //$NON-NLS-1$ //$NON-NLS-2$
+        column3.setWidth("20%", "30px"); //$NON-NLS-1$ //$NON-NLS-2$
+        DynamicTableColumn column4 = new DynamicTableColumn(table, SWT.NONE);
+        column4.setText(Resources.getMessage("ViewAttributeList.3")); //$NON-NLS-1$
+        column4.setWidth("15%", "30px"); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
The relative widths (%) of the 5 columns don't add up to 100%...

Reply:
Fixed. Please review once more.